task/sast-coverity-check/0.2/sast-coverity-check.yaml

```diff
@@ -171,8 +176,6 @@ spec:
         cpu: "1"
         memory: 1Gi
     env:
-    - name: BUILDAH_FORMAT
-      value: oci
```

Comment:
I may be wrong, but shouldn't this env variable be set to the value of `$(params.BUILDAH_FORMAT)` instead of being removed?

Reply:
As far as I can tell, the format isn't relevant to the coverity task. The fact that it gets the new parameter makes it confusing, but it's probably not a big deal. It's not the first parameter that it ignores and likely won't be the last (considering it inherits all params from buildah)

Reply:
Okay, in that case I won't hold up the PR for this, thanks for the info.

Reply:
This was removed everywhere because it doesn't matter what format is used when buildah runs to build since the format can just be specified when pushing.

I agree that it is confusing that coverity gets the new format though.